src/index.js

```diff
@@ -2,9 +2,10 @@ const path = require("path"),
 	https = require("https");
 
 const PullRequestRegex = /(\d+)\/merge/i,
-	MaxBranchNameLength = 20;
+	MaxBranchNameLength = 20,
+	namingConventionRegex = /^([A-Z]){2,3}-\d+.+$/i; //namingConventionRegex specifies 2 or 3 alphabetic characters, then a hyphen, then some numbers, then requires some more characters.
```

Comment:
This should follow PascalCaseNaming to be consistent with the other constants. Is `BranchNamingConventionRegex` a better name?

Also, maybe the regex should be something like `^[A-Z]{2,10}-\d+-[A-Z0-9-_]+$` - because Jira project keys are up to 10 chars by default, we want a dash after the issue id and we don't want random characters in the branch name,